addons/share-through-mystuff/addon.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "Share through My Stuff",
+  "description": "Share projects through the My Stuff page.",
+  "credits": [
+    {
+      "name": "Steve0Greatness",
+      "link": "https://scratch.mit.edu/users/Steve0Greatness/"
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "userstyle.css",
+      "matches": ["https://scratch.mit.edu/mystuff"]
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["https://scratch.mit.edu/mystuff"],
+      "runAtComplete": true
+    }
+  ],
+  "dynamicDisable": true,
+  "dynamicEnable": true,
+  "tags": ["community"],
```

I think you need to add `"versionAdded": "<VERSION>"` to the addon manifest in order for the check to pass.
```suggestion
  "tags": ["community"],
  "versionAdded": "1.30.0",
```